recipes/pycld2/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pycld2" %}
+{% set version = "0.31" %}
+{% set sha256 = "7d0f51b1550a6f964409f55f02b1f784d59a0d1b2718efa87c14d43344078403" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win64]
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - pycld2
+
+about:
+  home: https://github.com/aboSamoor/pycld2
+  license: APACHE
```

Comment:
This should be `Apache-2.0`

Reply:
I made this change to the meta.yaml file.